dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -4,10 +4,10 @@
   {
    "name": "page1",
    "elements": [
-     {
+    {
      "type": "html",
      "name": "clickTo44",
-     "visible": false,
+     "visible": true,
      "titleLocation": "hidden",
      "html": "<h2 style=\"background-color: #ffff00\"> Variable Test</h2>\n<ul>\n<li><span style=\"background-color: #ffff00;\">environment: {environment}</span></li>\n<li><span style=\"background-color: #ffff00;\">userId: {user_id}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopId: {workshop_id}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopCourse: {workshop_course}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopSubject: {workshop_subject}</span></li>\n<li><span style=\"background-color: #ffff00;\">regionalPartnerName: {regional_partner_name}</span></li>\n<li><span style=\"background-color: #ffff00;\">sessionId: {session_id}</span></li>\n</ul>"
```

Comment:
do we need environment, user_id, workshop_id and session_id? We aren't sending them as variables since we didn't need to fill them in anywhere

Reply:
This is just an artifact of copy over from jotform.  Not an expression of need for these variables.  And this is in a hidden field I only show for testing purposes. Nonetheless, I've removed and replaced with the applicable vars.